src/output.c

```diff
@@ -2984,7 +2984,7 @@ int clear_more(int new)
 	    nexttop(display_screen);
 	if (scrolled < new) {
 	    display_screen->paused = 0;
-	    display_screen->outcount = new - scrolled;
+            display_screen->outcount = new - scrolled;
```

Comment:
Whitespace issues?

Reply:
Good catch, Github's web-merge is apparently something to be careful with in that regard. Just pushed corrections.